### ISSUE TITLE
Quick Question

### ISSUE THREAD

**Issue by icariantk** [5axes/Cura-Postprocessing-Scripts#13]
Is there a way to show a popup menu to confirm with a yes or no if the script has to run? 

Tryed with QT6 but failed 

@IcarianOM at Telegram if this is not the correct way to ask this questions

**Comment by 5axes** [comment]
I don't think so. Postprocessing script are just script used to modify the postprocessed Gcode. So there are no mechanism for question or user sélection.
